commands/core/site_install.drush.inc

```diff
@@ -4,7 +4,7 @@ function site_install_drush_command() {
   $items['site-install'] = array(
     'description' => 'Install Drupal along with modules/themes/configuration using the specified install profile.',
     'arguments' => array(
-      'profile' => 'the install profile you wish to run. defaults to \'default\' in D6, \'standard\' in D7+',
+      'profile' => 'the install profile you wish to run. defaults to \'default\' in D6, \'standard\' in D7+, unless an install profile is marked as exclusive (or as a distribution in D8+ terminology) in which case that is used',
```

Comment:
Sorry to be a stickler, but help uses sentence case. So, starts with a capital and ends with a period. Also, could we add code comment that documents how an install profile is marked as a distribution or 'exclusive'?